activiti-examples/activiti-api-web-example/src/main/java/org/activiti/examples/DemoApplicationConfiguration.java

```diff
@@ -83,12 +88,10 @@ protected void configure(HttpSecurity http) throws Exception {
                 .authenticated()
                 .and()
                 .httpBasic();
-
-
     }
 
     @Bean
-    public PasswordEncoder passwordEncoder() {
+    public static PasswordEncoder passwordEncoder() {
         return new BCryptPasswordEncoder();
```

Comment:
IIRC adding a static would return a new `BCryptPasswordEncoder`  for every method call. Is this desired?

Reply:
Spring will instantiate this bean as singleton only once using static method. This is done to break circular dependency between bean and configuration class that uses it.